src/about.html

```diff
@@ -40,54 +40,54 @@ <h3>What are Existential Graphs?</h3>
         <p>
           Proposed by <cite><a href="https://en.wikipedia.org/wiki/Charles_Sanders_Peirce">Charles 
           Sanders Peirce</a></cite>, an <cite><a href="https://en.wikipedia.org/wiki/Existential_graph">
-          Existential Graph</a></cite> is a diagrammatic or visual representation of a logical expression. 
-          There are 3 types of Existential Graphs: Alpha Existential Graphs, Beta Existential Graphs and
-          Gamma Existential Graphs. In our application, we cover Alpha Existential Graphs.        
+          Existential Graph</a></cite> is a visual, also called diagrammatic, representation of some logical expression. 
+          There are three types of Existential Graphs: Alpha, Beta and Gamma.
+          In Peirce My Heart, we have implemented only Alpha Existential Graphs (AEGs).        
         </p>
 
-        <h4>The parts of an Alpha Existential Graph (AEG)</h4>
-        Alpha Existential Graphs are a form of Existential Graphs that are isomorphic to <cite>
-        <a href="https://en.wikipedia.org/wiki/Propositional_calculus">Propositional Logic</a></cite>
-        Propositional Logic. Graphs of this system can be used to construct entire propositional 
-        logic statements with just 3 components: 
+        <h4>The components of an Alpha Existential Graph</h4>
+        AEGs are isomorphic to <cite>
```

Comment:
AEG system is isomorphic to propositional logic, AEGs themselves are a way to represent propositional formulae.